SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_BlueberryCatchPhoto.h

```diff
@@ -0,0 +1,46 @@
+/*  Blueberry Quests
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#ifndef PokemonAutomation_PokemonSV_BlueberryCatchPhoto_H
+#define PokemonAutomation_PokemonSV_BlueberryCatchPhoto_H
+
+#include<vector>
+#include "PokemonSV/Programs/Farming/PokemonSV_BlueberryQuests.h"
+#include "PokemonSV/Options/PokemonSV_BBQOption.h"
+
+using namespace std;
+
+namespace PokemonAutomation{
+    class BotBaseContext;
+    class ConsoleHandle;
+    struct ProgramInfo;
+namespace NintendoSwitch{
+namespace PokemonSV{
+
+//Navigate to a photo target
+CameraAngle quest_photo_navi(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context, BBQOption& BBQ_OPTIONS, BBQuests& current_quest);
```

Comment:
You're not writing to `BBQuests& current_quest` right? If it's strictly an input parameter, take it by value since it's a primitive.